src/bun.js/bindings/Uint8Array.cpp

```diff
@@ -25,4 +22,23 @@ extern "C" JSC::EncodedJSValue JSUint8Array__fromDefaultAllocator(JSC::JSGlobalO
 
     return JSC::JSValue::encode(uint8Array);
 }
+
+extern "C" JSC::EncodedJSValue JSArrayBuffer__fromDefaultAllocator(JSC::JSGlobalObject* lexicalGlobalObject, uint8_t* ptr, size_t length)
+{
+
+    JSC::JSArrayBuffer* arrayBuffer;
+
+    if (LIKELY(length > 0)) {
+        RefPtr<ArrayBuffer> buffer = ArrayBuffer::createFromBytes({ ptr, length }, createSharedTask<void(void*)>([](void* p) {
```

Comment:
should we handle exceptions here?

Reply:
this line won't throw, it's a C++ ref counted ptr